packages/message-list/src/vaadin-message-list-mixin.js

```diff
@@ -175,4 +186,9 @@ export const MessageListMixin = (superClass) =>
       const index = this._messages.findIndex((e) => e.tabIndex === 0);
       return index !== -1 ? index : 0;
     }
+
+    /** @private */
+    __announceChanged(announceMessages) {
+      this.ariaLive = announceMessages ? 'polite' : null;
```

Comment:
Out of curiosity, what benefit does a separate property have over setting `ariaLive` manually?

We could also consider using [ElementInternals](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals/ariaLive) API in V25 if we want to have some default.

Reply:
The main cause was that especially in Flow we wanted a component-level flag so you don't need to access the Element API. Also the fact that the property should be applied to the host element is kind of a techincal detail and in theory might change.

Reply:
Yes, we originally thought that the attribute would need to be applied to the `<div role="list"...>` inside the shadow root. The root element _seems_ to suffice, but I'm not 100% convinced about it yet.